corehq/apps/repeaters/models.py

```diff
@@ -419,6 +419,7 @@ class RepeatRecordAttempt(DocumentSchema):
     cancelled = BooleanProperty(default=False)
     datetime = DateTimeProperty()
     failure_reason = StringProperty()
+    success_response = StringProperty()
```

Comment:
Yeah, I agree that'd be a better state, but it didn't seem super straightforward to implement.  `failure_reason` can also be a string version of an exception that happens, and it's stored on the repeater class in addition to the attempts.  Its presence is also used as a boolean in some cases (`if repeater.failure_reason:`), so that would also require a bit of a refactor.